main.go

```diff
@@ -15,63 +15,13 @@ import (
 	"io"
 	"io/ioutil"
 	"os"
-	"sort"
 	"strconv"
 	"strings"
-	"unicode"
 
-	"github.com/fatih/camelcase"
-	"github.com/fatih/structtag"
+	"github.com/fatih/gomodifytags/modifytags"
 	"golang.org/x/tools/go/buildutil"
 )
 
-// structType contains a structType node and it's name. It's a convenient
-// helper type, because *ast.StructType doesn't contain the name of the struct
-type structType struct {
-	name string
-	node *ast.StructType
-}
-
-// output is used usually by editors
-type output struct {
-	Start  int      `json:"start"`
-	End    int      `json:"end"`
-	Lines  []string `json:"lines"`
-	Errors []string `json:"errors,omitempty"`
-}
-
-// config defines how tags should be modified
-type config struct {
```

Comment:
We can leave the type definitions here, to minimize delta and preserve the original structure of the code as much as possible.